test-ssl.sh

```diff
@@ -66,7 +66,7 @@ DB_URL="$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' "$PG_CONTAI
 
 test_accept_cipher --tlsv1_2 "$DB_URL"
 
-if [[ "$IMG" =~ postgresql:13 ]]; then
+if [[ "$IMG" =~ postgresql:1[3-4] ]]; then
```

Comment:
For readability I'd prefer:

```suggestion
if [[ "$IMG" =~ postgresql:(13|14) ]]; then
```